tests/test_split_collect.py

```diff
@@ -0,0 +1,120 @@
+import glob
+import os
+import random
+import shutil
+import string
+import subprocess
+
+import pytest
+import sh
+
+from pipeline.translate.splitter import split_file
+
+COMPRESSION_CMD = "zstdmt"
+
+OUTPUT_DIR = "tests_data"
+
+
+@pytest.fixture(scope="function")
+def clean():
+    if os.path.exists(OUTPUT_DIR):
+        shutil.rmtree(OUTPUT_DIR)
+    os.makedirs(OUTPUT_DIR, exist_ok=True)
```

Comment:
This kind of code makes me nervous that if we have a programming error we'll obliterate a valid directory.

Suggestion: It would be nice to do a bit of light checking, like make sure `./data` exists first, and then do this in a sub-directory there.